tests/test_fabricpath_global.rb

```diff
@@ -37,11 +37,10 @@ def teardown
   def no_feature_fabricpath
     # Turn the feature off for a clean test.
     config('no feature-set fabricpath')
-    config('no install feature-set fabricpath')
   end
 
   def n5k_platform?
```

Comment:
sure, I will rename it to n5k6k_platforms (but the 6k is going to be re-named as a variant 5k itself in future from what I hear)